tests/generation/predictors/seq2seq_test.py

```diff
@@ -19,7 +19,8 @@ def test_uses_named_inputs_with_simple_seq2seq(self):
         predicted_tokens = result.get("predicted_tokens")
         assert predicted_tokens is not None
         assert isinstance(predicted_tokens, list)
-        assert all(isinstance(x, str) for x in predicted_tokens)
+        for predicted_token in predicted_tokens:
+            assert all(isinstance(x, str) for x in predicted_token)
```

Comment:
`predicted_tokens` is now a list of lists?

Reply:
Yes, it contains top n sequences, could see in [here](https://github.com/allenai/allennlp-models/pull/90#discussion_r453540252)